### PR TITLE
finish

### PR DESCRIPTION
**Pull request by smuneera253**
login

### CODE REVIEW

**Comment by smuneera253** [comment]
login

File: client-side(frontend)/capstone-project/public/flower5.png.crdownload
Comment (soumithaJtbb) [inline client-side(frontend)/capstone-project/public/flower5.png.crdownload:1]: remove this

File: client-side(frontend)/capstone-project/public/flower3.png
Comment (soumithaJtbb) [inline client-side(frontend)/capstone-project/public/flower3.png:1]: failed images - please remove this

File: client-side(frontend)/capstone-project/public/flower5.png.crdownload
Comment (smuneera253) [reply client-side(frontend)/capstone-project/public/flower5.png.crdownload:1]: i removed

File: client-side(frontend)/capstone-project/public/flower3.png
Comment (smuneera253) [reply client-side(frontend)/capstone-project/public/flower3.png:1]: i removed

**Pull request merged**